{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2018-16427",
      "ASSIGNER": "cve@mitre.org"
    },
    "affects": {
      "vendor": {
        "vendor_data": [

        ]
      }
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [

          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "https://github.com/OpenSC/OpenSC/pull/1447/commits/8fe377e93b4b56060e5bbfb6f3142ceaeca744fa",
          "name": "https://github.com/OpenSC/OpenSC/pull/1447/commits/8fe377e93b4b56060e5bbfb6f3142ceaeca744fa",
          "refsource": "MISC",
          "tags": [

          ]
        },
        {
          "url": "https://github.com/OpenSC/OpenSC/releases/tag/0.19.0-rc1",
          "name": "https://github.com/OpenSC/OpenSC/releases/tag/0.19.0-rc1",
          "refsource": "MISC",
          "tags": [

          ]
        },
        {
          "url": "https://www.x41-dsec.de/lab/advisories/x41-2018-002-OpenSC/",
          "name": "https://www.x41-dsec.de/lab/advisories/x41-2018-002-OpenSC/",
          "refsource": "MISC",
          "tags": [

          ]
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "Various out of bounds reads when handling responses in OpenSC before 0.19.0-rc1 could be used by attackers able to supply crafted smartcards to potentially crash the opensc library using programs."
        }
      ]
    }
  },
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [

    ]
  },
  "impact": {
  },
  "publishedDate": "2018-09-04T00:29Z",
  "lastModifiedDate": "2018-09-04T00:29Z"
}